docker-image/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.4
+FROM alpine:3.7
```

Comment:
Please can you copy the change to the ppc version?

Reply:
DOne